Class-2-(ControlFlow-&-Collections).md

```diff
@@ -0,0 +1,40 @@
+# Class 2: Control Flow and Collections
+
+## In the prep work this week, the students learned:
+
+1. The Java syntax for conditionals and loops:
+    1. if/else if/else
+    1. switch statements
+    1. for, while, and do...while
+1. How to use the Array, ArrayList, and HashMap collections.
+
+## Announcements
+
+1. Remind the students to continue working on Graded Assignment #1! By the end of class 2, they should be able to complete a significant portion of the project.
```

Comment:
Based on the schedule, they should be able to complete the whole assignment. Maybe switch it to "Remind students that Graded Assignment #1 is due soon! Students should be able to complete the project after today's class."

Reply:
Done. Thanks.